game.py

```diff
@@ -20,7 +21,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            #log correct letter guess
+            correct_letter_guess_statuses[user_input] = True
+            #check if complete
```

Comment:
From the [PEP 8 Style Guide](https://peps.python.org/pep-0008/#inline-comments): 

*"Use inline comments sparingly.*

*An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.*

*Inline comments are unnecessary and in fact distracting if they state the obvious..."*
```suggestion
            # check if complete
```